spec.html

```diff
@@ -122,6 +122,29 @@ <h1>
 		</emu-note>
 	</emu-clause>
 
+	<emu-clause id="sec-create-type-error-copy" type="abstract operation">
+		<h1>
+			CreateTypeErrorCopy (
+				_realmRecord_: a Realm Record,
+				_originalError_: an ECMAScript language value
+			)
+		</h1>
+		<emu-alg>
+			1. Let _newError_ be a newly created *TypeError* object.
+			1. NOTE: _newError_ is created in _realmRecord_.
+			1. If Type(_originalError_) is Object, then
+				1. Let _message_ be Completion(Get(_originalError_, "message")).
+				1. If _message_ is not an abrupt completion, then
+					1. If Type(_message_) is String, then
+						1. NOTE: Copying the string message when possible.
+						1. Perform CreateNonEnumerableDataPropertyOrThrow(_newError_, "message", _message_).
+			1. Return _newError_.
+		</emu-alg>
+		<emu-note>
+			For implementations that provide implementation-specific stack for introspection APIs, the TypeError object can be augmented.
```

Comment:
@littledan what do you think about this note?

Reply:
This note is unclear as to whether implementations can, or should, or must not, allow stack information from inside the ShadowRealm to be copied across.

Reply:
Agreed, this doesn't answer the question at all.